VideoStreaming/SubViews/Cells/VideoTableViewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  VideoTableViewCell.swift
+//  VideoStreaming
+//
+//  Created by Julian Llorensi on 17/09/2019.
+//  Copyright © 2019 ESPN. All rights reserved.
+//
+
+import UIKit
```

Comment:
There was a ticket for this work, remember that a PR should only have what is expected for the ticket itself. ;).